src/index.html

```diff
@@ -17,6 +17,10 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch">
+      <div class="stopwatch__minutes-hand"></div>
+      <div class="stopwatch__circle"></div>
+      <div class="stopwatch__seconds-hand"></div>
```

Comment:
Each element within the BEM block should follow the pattern `block__element`. It's good that you've followed this pattern, but make sure the names 'minutes-hand' and 'seconds-hand' accurately describe the elements and their functions within the 'stopwatch' block.